test/test_kaldi_dirs.py

```diff
@@ -120,13 +120,12 @@ def multi_channel_kaldi_dir():
             "lbi-1272-135031-0000_1": "ffmpeg -threads 1 -i nonexistent.wav -ar 16000 -map_channel 0.0.1  -f wav -threads 1 pipe:1 |",
         },
         "segments": {
-            "lbi-1272-135031-0000-A": "lbi-1272-135031-0000_0 0.0 10.885",
-            "lbi-1272-135031-0000-B": "lbi-1272-135031-0000_1 0.0 10.885",
+            "lbi-1272-135031-0000-A-0": "lbi-1272-135031-0000_0 0.0 10.885",
+            "lbi-1272-135031-0000-B-1": "lbi-1272-135031-0000_1 0.0 10.885",
         },
     }
 
 
-@pytest.mark.xfail(reason="multi file recordings not supported yet")
```

Comment:
@jtrmal This multi-channel recording test which was expected to fail earlier is now passing.